tpcx_bb/queries/q09/tpcx_bb_query_09_sql.py

```diff
@@ -0,0 +1,126 @@
+#
+
+import sys
+
+
+from blazingsql import BlazingContext
+from xbb_tools.cluster_startup import attach_to_cluster
+from dask_cuda import LocalCUDACluster
+from dask.distributed import Client
+import os
+
+from xbb_tools.utils import (
+    benchmark,
+    tpcxbb_argparser,
+    run_bsql_query,
+)
+
+cli_args = tpcxbb_argparser()
+
+
+@benchmark(
+    compute_result=cli_args["get_read_time"], dask_profile=cli_args["dask_profile"]
+)
+def read_tables(data_dir):
+    bc.create_table("store_sales", data_dir + "/store_sales/*.parquet")
+    bc.create_table("customer_address", data_dir + "/customer_address/*.parquet")
+    bc.create_table(
+        "customer_demographics", data_dir + "/customer_demographics/*.parquet"
+    )
+    bc.create_table("date_dim", data_dir + "/date_dim/*.parquet")
+    bc.create_table("store", data_dir + "/store/*.parquet")
+
+
+@benchmark(dask_profile=cli_args["dask_profile"])
+def main(data_dir, client):
+    read_tables(data_dir)
+
+    query = """
+        SELECT SUM(ss1.ss_quantity)
+        FROM store_sales ss1, date_dim dd,customer_address ca1 , store s ,customer_demographics cd
+        -- select date range
+        WHERE ss1.ss_sold_date_sk = dd.d_date_sk 
+        AND dd.d_year = 2001
+        AND ss1.ss_addr_sk = ca1.ca_address_sk
+        AND s.s_store_sk = ss1.ss_store_sk
+        AND cd.cd_demo_sk = ss1.ss_cdemo_sk
+        AND 
+        (
+            (
+                cd.cd_marital_status = 'M'
+                AND cd.cd_education_status = '4 yr Degree'
+                AND 100 <= ss1.ss_sales_price
+                AND ss1.ss_sales_price <= 150
+            ) 
+            OR 
+            (
+                cd.cd_marital_status = 'M'
+                AND cd.cd_education_status = '4 yr Degree'
+                AND 50 <= ss1.ss_sales_price
+                AND ss1.ss_sales_price <= 200
+            ) 
+            OR 
+            (
+                cd.cd_marital_status = 'M'
+                AND cd.cd_education_status = '4 yr Degree'
+                AND 150 <= ss1.ss_sales_price
+                AND ss1.ss_sales_price <= 200
+            )
+        ) 
+        AND 
+        (
+            (
+                ca1.ca_country = 'United States'
+                AND ca1.ca_state IN ('KY', 'GA', 'NM')
+                AND 0 <= ss1.ss_net_profit
+                AND ss1.ss_net_profit <= 2000
+            ) 
+            OR 
+            (
+                ca1.ca_country = 'United States'
+                AND ca1.ca_state IN ('MT', 'OR', 'IN')
+                AND 150 <= ss1.ss_net_profit
+                AND ss1.ss_net_profit <= 3000
+            ) 
+            OR 
+            (
+                ca1.ca_country = 'United States'
+                AND ca1.ca_state IN ('WI', 'MO', 'WV')
+                AND 50 <= ss1.ss_net_profit
+                AND ss1.ss_net_profit <= 25000
+            )
+        )
+    """
+
+    result = bc.sql(query)
+    result = result.rename(columns={"EXPR$0": "sum(ss_quantity)"})
```

Comment:
I believe this still introduces an additional copy versus `result.columns = [...]` with Dask, so if possible we would want to use `.columns`

Reply:
Updated as

**result.columns = ["sum(ss_quantity)"]**